src/component/Book.js

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
 import propTypes from 'prop-types';
 
-const Book = ({ book: { id, title, category } }) => (
-  <tr>
-    <td>{id}</td>
-    <td>{title}</td>
-    <td>{category}</td>
-  </tr>
-);
+const Book = ({ book, handler }) => {
+  const { id, title, category } = book;
+  return (
+    <tr>
+      <td>{id}</td>
+      <td>{title}</td>
+      <td>{category}</td>
+      <td><button type="button" id="{id}" onClick={() => handler({ id, title, category })}>remove</button></td>
```

Comment:
- [ ] [OPTIONAL] Why are you passing id, title and category to the removeBook action if you only need the book.id ?


Instead of writing this much you can simply pass the book object to the handler function, that would make your code shorter and more readable.

Ex:
`handler(book)`

Reply:
- [ ] [OPTIONAL] A cleaner way to implement this function on click would be to call it inside a handleClick function:

```
const Book = ({ book, handler }) => {
  const { id, title, category } = book;

  const handleClick = () => {
     handler(book);
  }

  return (
    ...
      
      <td><button type="button" id="{id}" onClick={handleClick}>remove</button></td>
```